tests/regress.sh

```diff
@@ -79,7 +79,12 @@ variable_body()
 
 keep_system_libs_body()
 {
-	export PKG_CONFIG_PATH="${selfdir}/lib1" LIBRARY_PATH="/test/local/lib"
+	export PKG_CONFIG_PATH="${selfdir}/lib1"
+	if [ "$(uname -s)" = Haiku ]; then
```

Comment:
i don't like this approach, but since it's one test, I guess it's fine.  would prefer to handle Haiku in the same way as the other environments (see `test_env.sh.in`)